server/odc-service/src/main/java/com/oceanbase/odc/service/common/DruidMonitor.java

```diff
@@ -0,0 +1,646 @@
+            this.defaultValue = defaultValue;
+        }
+    }
+
+    public static final class DruidSQLStateConstants {
+        public static final String EXECUTE_AND_RESULT_SET_HOLD_TIME = "ExecuteAndResultSetHoldTime";
+        public static final String EFFECTED_ROW_COUNT_HISTOGRAM = "EffectedRowCountHistogram";
+        public static final String LAST_ERROR_MESSAGE = "LastErrorMessage";
+        public static final String HISTOGRAM = "Histogram";
+        public static final String INPUT_STREAM_OPEN_COUNT = "InputStreamOpenCount";
+        public static final String BATCH_SIZE_TOTAL = "BatchSizeTotal";
+        public static final String FETCH_ROW_COUNT_MAX = "FetchRowCountMax";
+        public static final String ERROR_COUNT = "ErrorCount";
+        public static final String BATCH_SIZE_MAX = "BatchSizeMax";
+        public static final String LAST_ERROR_TIME = "LastErrorTime";
+        public static final String READER_OPEN_COUNT = "ReaderOpenCount";
+        public static final String EFFECTED_ROW_COUNT_MAX = "EffectedRowCountMax";
+        public static final String LAST_ERROR_CLASS = "LastErrorClass";
+        public static final String IN_TRANSACTION_COUNT = "InTransactionCount";
+        public static final String LAST_ERROR_STACK_TRACE = "LastErrorStackTrace";
+        public static final String RESULT_SET_HOLD_TIME = "ResultSetHoldTime";
+        public static final String TOTAL_TIME = "TotalTime";
+        public static final String CONCURRENT_MAX = "ConcurrentMax";
+        public static final String RUNNING_COUNT = "RunningCount";
+        public static final String FETCH_ROW_COUNT = "FetchRowCount";
+        public static final String MAX_TIME_SPAN_OCCUR_TIME = "MaxTimespanOccurTime";
+        public static final String LAST_SLOW_PARAMETERS = "LastSlowParameters";
+        public static final String READ_BYTES_LENGTH = "ReadBytesLength";
+        public static final String DB_TYPE = "DbType";
+        public static final String DATASOURCE = "DataSource";
+        public static final String SQL = "SQL";
+        public static final String LAST_ERROR = "LastError";
+        public static final String MAX_TIME_SPAN = "MaxTimespan";
+        public static final String BLOB_OPEN_COUNT = "BlobOpenCount";
+        public static final String EXECUTE_COUNT = "ExecuteCount";
+        public static final String EFFECTED_ROW_COUNT = "EffectedRowCount";
+        public static final String READ_STRING_LENGTH = "ReadStringLength";
+        public static final String EXECUTE_AND_RESULT_HOLD_TIME_HISTOGRAM = "ExecuteAndResultHoldTimeHistogram";
+        public static final String FILE = "File";
+        public static final String CLOB_OPEN_COUNT = "ClobOpenCount";
+        public static final String LAST_TIME = "LastTime";
+        public static final String FETCH_ROW_COUNT_HISTOGRAM = "FetchRowCountHistogram";
+
+        private DruidSQLStateConstants() {}
+    }
+
+    @Data
+    public static class DruidSQLStateVO {
```

Comment:
VO,DTO... such a name is not recommended in odc, how about remove 'VO' suffix?(DruidSqlState)

and, is 'public' necessary?